compile_Out:
0 errors
__SECTION__
compile_Err:

__SECTION__
simulation_Out:

__SECTION__
simulation_Err:
Traceback (most recent call last):
  File "C:\Users\lucco\Desktop\tests\Lang\compiler\vm_simulator.py", line 22, in <module>
    opName = opcodes[opcode].name
KeyError: 100